.mailmap

```diff
@@ -0,0 +1,2804 @@
+Aakash Jain <aakash_jain@apple.com> <aakash_jain@apple.com>
+Aakash Jain <aakash_jain@apple.com> <aakash_jain@apple.com@268f45cc-cd09-0410-ab3c-d52691b4dbfc>
```

Comment:
I realise `validate-committer-lists`produces this output, but this is almost certainly pointless; we only have very few commits from anyone with an SVN UUID:

```
   2 carlosgc@webkit.org@268f45cc-cd09-0410-ab3c-d52691b4dbfc
   2 ntim@apple.com@268f45cc-cd09-0410-ab3c-d52691b4dbfc
   4 fred.wang@free.fr@268f45cc-cd09-0410-ab3c-d52691b4dbfc
  12 zandobersek@gmail.com@268f45cc-cd09-0410-ab3c-d52691b4dbfc
  56 zan@falconsigh.net@268f45cc-cd09-0410-ab3c-d52691b4dbfc
  80 eocanha@igalia.com@268f45cc-cd09-0410-ab3c-d52691b4dbfc
 304 commit-queue@webkit.org@268f45cc-cd09-0410-ab3c-d52691b4dbfc
```

And the last of these was in June 2022.